torch/csrc/jit/mobile/import.cpp

```diff
@@ -230,6 +218,11 @@ class BytecodeDeserializer final {
   c10::IValue readArchive(
       const std::string& archive_name,
       std::shared_ptr<mobile::CompilationUnit> mcu);
+  void parseFunctionSchema(
```

Comment:
Add a new line above?

Reply:
Looks like it's the format convention: no new line between function declarations.